Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,445 @@
+                >
+                    <Center fontSize='xs' h='100%'>
+                        Angular Rate (deg/s)
+                    </Center>
+                </GridItem>
+
+                <GridItem
+                    rowStart={2}
+                    colStart={2}
+                    borderTop='1px'
+                    borderLeft='1px'
+                >
+                    <Center as='b' fontSize='1.5vh' h='100%'>
+                        {props.data?.linear_accel_x[0].toFixed(0) ?? -1.0}
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={2}
+                    colStart={3}
+                    borderTop='1px'
+                    borderLeft='1px'
+                >
+                    <Center as='b' fontSize='1.5vh' h='100%'>
+                        {props.data?.linear_accel_y[0].toFixed(0) ?? -1.0}
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={2}
+                    colStart={4}
+                    borderTop='1px'
+                    borderLeft='1px'
+                >
+                    <Center as='b' fontSize='1.5vh' h='100%'>
+                        {props.data?.linear_accel_z[0].toFixed(0) ?? -1.0}
+                    </Center>
+                </GridItem>
+
+                <GridItem
+                    rowStart={3}
+                    colStart={2}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderBottom='1px'
+                >
+                    <Center as='b' fontSize='1.5vh' h='100%'>
+                        {props.data?.angular_rate_roll[0].toFixed(0) ?? -1.0}
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={3}
+                    colStart={3}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderBottom='1px'
+                >
+                    <Center as='b' fontSize='1.5vh' h='100%'>
+                        {props.data?.angular_rate_pitch[0].toFixed(0) ?? -1.0}
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={3}
+                    colStart={4}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderBottom='1px'
+                >
+                    <Center as='b' fontSize='1.5vh' h='100%'>
+                        {props.data?.angular_rate_yaw[0].toFixed(0) ?? -1.0}
+                    </Center>      
+                </GridItem>
+            </Grid>
+            
+            <Grid
+                flex='1'
+                templateRows="1fr 1fr" 
+                templateColumns="1fr 1fr 1fr 1fr 1fr"
+                h='25%'
+                pt = '1'
+                pb = '1'
+                pl = '1'
+                pr = '1'
+            >
```

Comment:
```suggestion
            <Grid
                flex='1'
                templateRows="1fr 1fr" 
                templateColumns="1fr 1fr 1fr 1fr 1fr"
                h='25%'
                pt = '1'
                pb = '1'
                pl = '1'
                pr = '0.25'
            >
```
Again, fits better on my end.